aspnet/publishing/linuxproduction.rst

```diff
@@ -0,0 +1,190 @@
+Publish to a Linux Production Environment
+=========================================
+
+By `Sourabh Shirhatti`_
+
+In this guide, we will cover setting up a production-ready ASP.NET environment on an Ubuntu 14.04 Server. Though we will cover the specifics of using Ubuntu 14.04, the content will be applicable to other flavors of Linux.
+
+We will take an existing ASP.NET 5 application and place it behind a reverse-proxy server. We will then setup the reverse-proxy server to forward requests to our Kestrel web server.
+
+Additionally we will write a startup script to ensure our web application runs on startup as a daemon as well as a configure a process management tool to help restart our web application in the event of a crash to guarantee high availibility.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+Prerequisites
+-------------
+
+1. Acess to an Ubuntu 14.04 Server with a standard user account with
+   sudo privilege.
+2. An exisiting ASP.NET 5 application.
+
+Before getting started you require LibUV. Follow the instructions on the :doc:`/getting-started/installing-on-linux` page.
+
+Copy over your app
+------------------
+
+Run ``dnu publish --runtime dnx-coreclr-linux-x64.1.0.0-rc1-update1`` from your dev environment to package your
+application into a self-contained directory that can be run on your server.
+
+Before we proceed, go ahead and copy over your ASP.NET 5 application to your server using whatever tool (SCP, FTP, etc) integrates into your workflow. Try and run the app and navigate to ``http://<serveraddress>:<port>`` in your browser to see if the application runs fine on Linux. I recommend you have a working app before proceeding.
```

Comment:
I don't think the getting started pages include all necessary information.

14.04 is not enough to run an app published through `dnu publish`. You need libunwind, libicu and possibly others. I have these dependencies declared in a docker container that I use to publish *exclusively* the output of the publish commands (meaning no compile-time-only dependencies), if you are interested.